src/CommonItilObject_Item.php

```diff
@@ -1576,4 +1577,36 @@ public static function dropdownAllDevices(
         }
         return $rand;
     }
+
+    /**
+     * ITIL tabs for generic assets should only be displayed if the asset already
+     * has associated ITIL items OR if the current user profile is allowed to
+     * link this asset to ITIL items
+     *
+     * @param Asset $asset
+     *
+     * @return bool
+     */
+    protected function shouldDisplayTabForAsset(Asset $asset): bool
```

Comment:
IMHO, this change should be discussed in a separate PR. It is probably less consensual than you expected.

Reply:
I asked for confirmation to @orthagh and he told me this behavior is the correct one to apply.

In fact, it should also be done on basic assets (computer, monitors, ...) in another PR.